lib/gateways/notification/govNotify.js

```diff
@@ -0,0 +1,28 @@
+module.exports = ({ client, log }) => {
+  function send(dropbox, requestedBy, requestedByEmail) {
+    try {
+      log.info('Sending email notification to', {
+        requestedBy
+      });
+
+      const personalisation = {
+        name: requestedBy,
+        residentName: dropbox.customerName,
+        description: dropbox.description,
+        linkToDropbox: `${process.env.URL_PREFIX}/dropboxes/${dropbox.id}/view`
+      };
+
+      return client
+        .sendEmail(process.env.GOV_NOTIFY_TEMPLATE_ID, requestedByEmail, {
+          personalisation,
+          reference: null
+        })
+        .then(response => console.log(response))
+        .catch(err => console.error(err));
```

Comment:
This will return straight away, potentially _before_ the email has been sent. Maybe wrap this in a promise so that it can be awaited elsewhere?

```suggestion
      return new Promise((resolve, reject) => {
        client
          .sendEmail(process.env.GOV_NOTIFY_TEMPLATE_ID, requestedByEmail, {
            personalisation,
            reference: null
          })
          .then(response => resolve(response))
          .catch(err => { console.error(err); reject(err); });
        });
```